### PR TITLE
v1.9.0 - Privacy policy and f-trak docs

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Privacy Policy content page added
- Added documentation for `t-trak` module

### Changed
- Tidied up some of the index pages

### CODE REVIEW

File: docs/src/templates/pages/styleguide/js-components/trak.hbs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+<button data-trak='{
+    "trakEvent": "click",
+    "event": "My Event Descriptor"
+    "category": "My Category Descriptor",
+    "action": "My Action Descriptor",
+    "label": "My Label Descriptor"
+'>Submit</button>
+```
+
+The above attribute tells `f-trak` to fire a tracking event with the button is clicked, with the values specified.  For a full list of available parameter values that you can pass (including how to define custom values), then see the [trak parameters](trak-parameters) section below.
```
Comment (DamianMullins) [inline docs/src/templates/pages/styleguide/js-components/trak.hbs]: Should be `#trak-parameters` as the href?

File: package.json [hunk 2]
```diff
@@ -45,6 +46,10 @@
     "predeploy": "yarn --ignore-optional",
     "deploy": "gulp docs:deploy"
   },
+  "resolutions": {
```
Comment (DamianMullins) [inline package.json]: Have just read up on this property, very interesting. Haven't seen it before 💠

File: package.json [hunk 3]
```diff
@@ -20,21 +20,22 @@
...
-    "@justeat/f-footer": "^0.18.2",
-    "@justeat/f-header": "^0.26.0",
+    "@justeat/f-footer": "^0.21.0",
+    "@justeat/f-header": "^0.27.0",
     "@justeat/f-icons": "^0.8.0",
     "@justeat/f-toggle": "^0.13.0",
     "@justeat/f-validate": "^0.18.0",
-    "@justeat/fozzie": "^0.42.2",
+    "@justeat/fozzie": "^0.44.0",
+    "gift": "^0.10.2",
```
Comment (DamianMullins) [inline package.json]: What does this package do?

File: docs/src/templates/pages/styleguide/ui-templates/index.hbs [hunk 4]
```diff
@@ -8,11 +8,8 @@ navsub: ui-templates
 navactive: index
 ---
 
-<div class="sg-index">
-
-- [Example Content Page (e.g. Cookie Policy)]({{ baseUrl }}/styleguide/ui-templates/content-pages/)
-
-</div>
+- [Content Page – Cookie Policy]({{ baseUrl }}/styleguide/ui-templates/content-pages/)
```
Comment (DamianMullins) [inline docs/src/templates/pages/styleguide/ui-templates/index.hbs]: Feel like the Cookie Policy page shouldn't be at the root of this path?

File: package.json (hunk 2 above)
Comment (kevinrodrigues) [reply package.json]: Got a link?

File: package.json (hunk 3 above)
Comment (kevinrodrigues) [reply package.json]: Looks like it's some sort of cli tool for git but I don't see it being used it here.

File: package.json (hunk 2 above)
Comment (DamianMullins) [reply package.json]: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

File: package.json (hunk 3 above)
Comment (DamianMullins) [reply package.json]: Is it worth removing if it's not in use?

File: package.json (hunk 2 above)
Comment (ashleynolan) [reply package.json]: Yeh, it's super useful – I found it became an issue when I had updated to use the new colours in Fozzie, but it was getting stuck on the older version due to the includes in the header and footer, throwing Sass errors around variables that didn't exist.

This let's you override the depended version if you know that it won't break anything else by doing so :)

**Pull request merged**